src/Tizen.NUI.WindowSystem/src/public/EdgeDragEventArgs.cs

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright(c) 2023 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+namespace Tizen.NUI.WindowSystem
+{
+    /// <summary>
+    /// This class contains the data related to the EdgeDrag event.
+    /// </summary>
+    public class EdgeDragEventArgs
```

Comment:
✔️ DO use [System.EventArgs](https://learn.microsoft.com/en-us/dotnet/api/system.eventargs) or its subclass as the type of the second parameter of the event handler, and call it e.

Please refer to https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/event.